ui-community/src/components/layouts/admin/section-layout.container.tsx

```diff
@@ -12,19 +14,40 @@ interface SectionLayoutContainerProps {
 export const SectionLayoutContainer = (props: SectionLayoutContainerProps) => {
   const params = useParams();
 
-  const { data: memberData, loading: memberLoading, error: memberError } = useQuery(SectionLayoutContainerMemberByIdQueryDocument, {
-    variables: {
-      memberId: params.memberId ?? '',
-    }
-  });
+  const [memberQuery] = useLazyQuery(SectionLayoutContainerMemberByIdQueryDocument);
+  const [memberData, setMemberData] = useState<any>(null);
+  const [memberError, setMemberError] = useState<any>(null);
+  const [memberLoading, setMemberLoading] = useState<any>(null);
+
+  useEffect(() => {
+    const getData = async () => {
+      try{
+        const {
+          data: memberDataTemp,
+          loading: memberLoadingTemp,
+          error: memberErrorTemp
+        } = await memberQuery({
+          variables: {
+            memberId: params.memberId ?? ''
+          }
+        });
+        setMemberData(memberDataTemp);
```

Comment:
please inspect the "success" and errorMessage fields in the data to show error (if any)